.github/ISSUE_TEMPLATE/OOB-Deploy-Request.md

```diff
@@ -26,15 +26,19 @@ Fill out Details of need and the PRs related to OOB sections below.
    > Add Details Here ... 
    
 ## Verify The following
- - [ ] The OOB Deploy Requests is after the 2pm EST cutoff for regular deploy - [Deployment Policy](https://vfs.atlassian.net/wiki/spaces/SUP/pages/1279787009/Deployment+Policies) and [Deployment Schedules](https://depo-platform-documentation.scrollhelp.site/developer-docs/Deployments.1844641889.html)
- - [ ] The OOB Deploy Requests is critial and must be resolved before the next automated deploy.
+ - [ ] The OOB Deploy Request is after the 2pm EST cutoff for regular deploy.[^1] 
+ - [ ] The OOB Deploy Request is critial and must be resolved before the next automated deploy.
+ - [ ] You are prepared to create an Incident Post Mortem[^2] within two business days.
+
+[^1]: (See [Deployment Policy](https://vfs.atlassian.net/wiki/spaces/SUP/pages/1279787009/Deployment+Policies) and [Deployment Schedules](https://depo-platform-documentation.scrollhelp.site/developer-docs/Deployments.1844641889.html))
```

Comment:
Love the footnotes!

I realized the Deployment Policy links to Confluence, which VFS teams don't have access to. It links to the Confluence version of this page on the platform website: https://depo-platform-documentation.scrollhelp.site/developer-docs/deployment-policies Can you link this one for Deployment Policy instead?